src/context/userContext/UserContext.tsx

```diff
@@ -2,12 +2,23 @@ import React, { FC, useContext, useEffect, useState } from 'react';
 import { User } from '../../types/models';
 import { useAuth } from '../authContext/AuthContext';
 import { useApi } from '../apiContext/ApiContext';
+import { daysAgo } from '../../helpers/time';
+
+type EmailNotVerifiedInfo = {
+  isVerified: true;
+} | {
+  email: string;
+  isVerified: false;
+  isRegisteredRecently: boolean;
+}
 
 type UserProviderContext = {
   isAdmin: boolean;
   isMentor: boolean;
   isLoading: boolean;
   currentUser?: User;
+  emailVerifedInfo?: EmailNotVerifiedInfo;
```

Comment:
Thank you!
If this is the only issue with the changes I made in the auth0 area, I'm happy ;)

Reply:
The changes made regarding Auth0 are limited I would say. But yeah, looks good!

Reply:
Thanks!
I mostly interested in a feedback about the verification email process.
If you can take a look on the [server PR](https://github.com/Coding-Coach/find-a-mentor-api/pull/249) too it would be great.

Reply:
Looks good 🧙

Reply:
Thanks!